scripts/release/debian/deb_install.sh

```diff
@@ -13,7 +13,7 @@
 # well-known publishers.                                                                                              #
 #######################################################################################################################
 
-set -e
+set -e -o pipefail
```

Comment:
Similar to https://github.com/MicrosoftDocs/azure-docs-cli/pull/4106#discussion_r1555323722. I don't think we should change it without absolute necessity.